pages/administration/tools/booking-lookup.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <AdminPage title="Bookings Lookup">
+    <div class="flex items-end space-x-4">
+      <div>
+        <label>User</label>
+        <UiInputText
+            v-model="userSearch"
+                     placeholder="Filter by user" />
+      </div>
```

Comment:
Does this search by the user who owns the booking, or the user who made the booking? Should we have some toggle to easily switch between the two to provide more functionality?

Reply:
Currently this searches by the owner, but I will add a toggle to switch between the two